protobuf/lib/src/protobuf/coded_buffer_writer.dart

```diff
@@ -341,7 +341,7 @@ class CodedBufferWriter {
             value is Uint8List ? value : Uint8List.fromList(value));
         break;
       case PbFieldType._STRING_BIT:
-        _writeBytesNoTag(_utf8.encoder.convert(value));
+        _writeBytesNoTag(const Utf8Encoder().convert(value));
```

Comment:
You can then remove the `_utf8` codec in:
```
protobuf/lib/protobuf.dart:62:const _utf8 = Utf8Codec(allowMalformed: true);
```

Reply:
It's already removed, see above.